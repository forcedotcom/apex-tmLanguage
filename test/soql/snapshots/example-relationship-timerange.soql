SELECT UserId, COUNT(Id) FROM LoginHistory WHERE LoginTime > 2010-09-20T22:16:30.000Z AND LoginTime < 2010-09-21T22:16:30.000Z GROUP BY UserId
