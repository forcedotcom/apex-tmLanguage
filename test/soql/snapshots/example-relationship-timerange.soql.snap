>SELECT UserId, COUNT(Id) FROM LoginHistory WHERE LoginTime > 2010-09-20T22:16:30.000Z AND LoginTime < 2010-09-21T22:16:30.000Z GROUP BY UserId
#^^^^^^ source.soql keyword.operator.query.select.apex
#      ^ source.soql
#       ^^^^^^ source.soql keyword.query.field.apex
#             ^ source.soql punctuation.separator.comma.apex
#              ^ source.soql
#               ^^^^^ source.soql support.function.query.apex
#                    ^ source.soql punctuation.parenthesis.open.apex
#                     ^^ source.soql keyword.query.field.apex
#                       ^ source.soql punctuation.parenthesis.close.apex
#                        ^ source.soql
#                         ^^^^ source.soql keyword.operator.query.from.apex
#                             ^ source.soql
#                              ^^^^^^^^^^^^ source.soql storage.type.apex
#                                          ^ source.soql
#                                           ^^^^^ source.soql keyword.operator.query.where.apex
#                                                ^ source.soql
#                                                 ^^^^^^^^^ source.soql keyword.query.field.apex
#                                                          ^ source.soql
#                                                           ^ source.soql keyword.operator.relational.apex
#                                                            ^ source.soql
#                                                             ^^^^^^^^^^^^^^^^^^^^^^^^ source.soql constant.numeric.datetime.apex
#                                                                                     ^ source.soql
#                                                                                      ^^^ source.soql keyword.operator.query.apex
#                                                                                         ^ source.soql
#                                                                                          ^^^^^^^^^ source.soql keyword.query.field.apex
#                                                                                                   ^ source.soql
#                                                                                                    ^ source.soql keyword.operator.relational.apex
#                                                                                                     ^ source.soql
#                                                                                                      ^^^^^^^^^^^^^^^^^^^^^^^^ source.soql constant.numeric.datetime.apex
#                                                                                                                              ^ source.soql
#                                                                                                                               ^^^^^^^^ source.soql keyword.operator.query.apex
#                                                                                                                                       ^ source.soql
#                                                                                                                                        ^^^^^^ source.soql keyword.query.field.apex
>